tests/inline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2347,6 +2347,58 @@ try {
 ================================================================================
 `;
 
+exports[`mixed-54.php 1`] = `
+====================================options=====================================
+parsers: ["php"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<?php
+/**
+ * @var string $a
+ */
+?>
+<?= $a ?>
+<?php echo $a; ?>
+
+=====================================output=====================================
+<?php
+/**
+ * @var string $a
+ */
+?>
+<?= $a ?>
+<?php echo $a; ?>
+
+================================================================================
+`;
+
+exports[`mixed-55.php 1`] = `
+====================================options=====================================
+parsers: ["php"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<?php
+/**
+ * @var string $a
+ */
+?>
+<?php echo $a; ?>
+<?= $a ?>
+
+=====================================output=====================================
+<?php
```

Comment:
Can we add test(s) where we start it with `<?=`?

Reply:
added `mixed-56.php`